atuin-client/src/import/resh.rs

```diff
@@ -122,18 +122,18 @@ impl Importer for Resh {
                 difference.num_nanoseconds().unwrap_or(0)
             };
 
-            h.push(History {
-                id: uuid_v7().as_simple().to_string(),
-                timestamp,
-                duration,
-                exit: entry.exit_code,
-                command: entry.cmd_line,
-                cwd: entry.pwd,
-                session: uuid_v7().as_simple().to_string(),
-                hostname: entry.host,
-                deleted_at: None,
-            })
-            .await?;
+            let imported = History::import()
+                .interpreter(Self::NAME)
+                .command(entry.cmd_line)
+                .timestamp(timestamp)
+                .duration(duration)
+                .exit(entry.exit_code)
+                .cwd(entry.pwd)
+                .hostname(entry.host)
+                // CHECK: should we add uuid here? It's not set in the other importers
```

Comment:
I left it as it was, but added the comment to highlight, that we don't set `session_id` in other importers.
So in this case all entries, imported from `resh` will have different `session_id`.